src/Converter.php

```diff
@@ -146,8 +146,8 @@ public function convert($data = null): ?array
 			$cfg = $this->getClassMappingConfigOrThrow($data);
 			$worker = new $cfg[ RB::KEY_HANDLER ]();
 			$result = $worker->convert($data, $cfg);
-			$result = empty($cfg[ RB::KEY_KEY ]) ? $result : [$cfg[ RB::KEY_KEY ] => $result];
-		}
+			$result = is_null($cfg[ RB::KEY_KEY ]) ? $result : [$cfg[ RB::KEY_KEY ] => $result];
```

Comment:
please do not use `is_null()` but do regular comparision: `$cfg[ RB::KEY_KEY ] === null`